torch/_meta_registrations.py

```diff
@@ -48,7 +47,6 @@ def toRealValueType(dtype):
     }
     return from_complex.get(dtype, dtype)
 
-
```

Comment:
don't wobble lines unnecessarily; especially in a long lived branch these lead to spurious merge conflicts. Big problem!